custom/stdweb/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "getrandom-stdweb"
+version = "0.1.0"
+edition = "2018"
+authors = ["The Rand Project Developers"]
+license = "MIT OR Apache-2.0"
+description = "Custom shim for using getrandom with stdweb"
+documentation = "https://docs.rs/getrandom-stdweb"
+repository = "https://github.com/rust-random/getrandom/tree/master/custom/stdweb"
+categories = ["wasm"]
+
+[dependencies]
+getrandom = { path = "../..", features = ["custom"] }
+stdweb = "0.4.18"
+log = "0.4"
+
+# Unstable features used to build tests correctly
+[features]
+default = ["test-stdweb"]
+test-stdweb = []
+
+[[test]]
+name = "common"
+path = "../../tests/common.rs"
```

Comment:
Do we really need it? IIUC we already test `stdweb` and `wasm-bindgen` via `getrandom`. Or is it for removing WASM-related test features to the custom crates?

Reply:
We do need this. When we move the wasm/stdweb functionality to Custom RNGs, they can no longer be tested by `getrandom` main test code. These `[[test]]`s simply  restore the old testing behavior.